src/main/resources/eu/europa/ted/eforms/sdk/analysis/drools/fieldsAndNodesRules.drl

```diff
@@ -99,16 +99,28 @@ then
   results.add(new ValidationResult($f, "Privacy code " + $privacyCode + " is not in the codelist " + $codelistId, ValidationStatusEnum.ERROR));
 end
 
-// TEDEFO-2205: Xpath absolute uniqueness is verified.
-//  For fields.json file:
-//    The xpathAbsolute of a field should not be equal to the xpathAbsolute of a node
-rule "Xpath absolute uniqueness is verified"
+rule "Absolute XPath of field is not equal to the absolute XPath of a node"
 when
-  /fields[ $f: this, $fieldId : id, $xpathAbsoluteField: xpathAbsolute ];
-  /nodes[ $n: this, $nodeId : id, $xpathAbsoluteNode: xpathAbsolute ]
-  eval($xpathAbsoluteField == $xpathAbsoluteNode)
+  /fields[ $f: this, $xpathAbsoluteField: xpathAbsolute ];
+  /nodes[ $nodeId : id, xpathAbsolute == $xpathAbsoluteField ]
 then
-  results.add(new ValidationResult($f, "xpath absolute are equal for field " + $fieldId + ", node " + $nodeId, ValidationStatusEnum.ERROR));
+  results.add(new ValidationResult($f, "Absolute XPath is the same as for node " + $nodeId, ValidationStatusEnum.ERROR));
+end
+
+rule "Absolute XPath of field is not equal to the absolute XPath of another field"
+when
+  /fields[ $f: this, $xpathAbsoluteField: xpathAbsolute, !id.startsWith("OPA-") ];
+  /fields[ $otherFieldId : id, id < $f.id, !id.startsWith("OPA-"), xpathAbsolute == $xpathAbsoluteField ]
+then
+  results.add(new ValidationResult($f, "Absolute XPath is the same as for field " + $otherFieldId, ValidationStatusEnum.ERROR));
+end
+
+rule "Absolute XPath of node is not equal to the absolute XPath of another node"
+when
+  /nodes[ $n: this, $xpathAbsoluteField: xpathAbsolute ];
```

Comment:
Not technically required but $xpathAbsoluteField is should be $xpathAbsoluteNode in this one

Reply:
Fixed in added commit.